### PR TITLE
Refactor Telemetry tag

### PR DESCRIPTION
**Pull request by katevoss**
Closes https://github.com/catalpainternational/canoe/issues/97

This is marked as a draft to get early feedback on the PR. Then I'll continue with improvements suggested in https://github.com/catalpainternational/canoe/issues/97

### CODE REVIEW

File: src/js/Telemetry.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    logPageView,
+    logNotificationReceived,
+    logClickedPlayOnVideo,
+    logATestAnswer,
+    setGoogleAnalyticsGlobalDimensions
+} from 'js/GoogleAnalytics';
+import { ON_PLAY_VIDEO, ON_ANSWERED_TEST_QUESTION } from 'js/Events';
+
+export function setTelemetry() {
+    setGoogleAnalyticsGlobalDimensions();
```
Comment (PeteCoward) [inline src/js/Telemetry.js:11]: I think this is a no-op function, as we now set those dimensions in every call to post an event. If so we can remove it

**Comment by PeteCoward** [comment]
looks good @katevoss 
Let me know if you have questions about going further or make it non-draft for full review.

We could even go further and fully remove this file fully, 
1. moving the video and test answer events to direct function calls
2 including the hashchange event call in the routing code
3. moving the notification event into the notifications code

**Comment by sandroroux** [comment]
@PeteCoward, if you're suggesting that we remove `GoogleAnalytics.js` and spread out its methods, I disagree.

Similar code belongs in the same place. Let's preserve that coherence. It's important for long-term maintainability.

**Comment by katevoss** [comment]
@PeteCoward Here is the change to the service worker you described from earlier https://github.com/catalpainternational/canoe/pull/100/commits/8b7c00ca940b06c438f5e09b613786c4cb46f377

**Comment by katevoss** [comment]
Closing this PR as it has been merged into https://github.com/catalpainternational/canoe/pull/99/

**Pull request closed without merging**